packages/service/src/storybook/launcher.ts

```diff
@@ -78,8 +78,12 @@ export default class VisualLauncher extends BaseClass  {
                 storybookUrl,
             })
 
+            // Clear the current capabilities
+            capabilities.length = 0
+            log.info('Clearing the current capabilities.')
+
             // Create the capabilities
-            createStorybookCapabilities(capabilities, log)
+            createStorybookCapabilities(capabilities as WebdriverIO.Capabilities[], log)
```

Comment:
What happened if the user runs Visual Service with Multiremote capabilities? Wouldn't this somewhere throw an error?

Reply:
@christian-bromann 

The are cleaned [here](https://github.com/webdriverio/visual-testing/blame/d4a49e22bb11d7f0da1e884dc3dc75d0d93fa715/packages/service/src/storybook/launcher.ts#L82), or am I doing something wrong?

Reply:
`capabilities` can be an object when people use multiremote, e.g.

```ts
capabilities: {
  browserA: { capabilities: { browserName: 'chrome' } },
  browserb: { capabilities: { browserName: 'firefox' } },
}
```

Setting `.length` to `0` doesn't sound like it would resolve the issue. I suggest that we check whether non-multiremote caps are used and throw an error otherwise.

Reply:
Is there a better way to overwrite the caps for all? I don't want to throw an error because we should just overwrite the caps

Reply:
You can take an example from the Sauce service: https://github.com/webdriverio/webdriverio/blob/658c217ecf224a5ba66394e0109747cc117f1b81/packages/wdio-sauce-service/src/launcher.ts#L66-L85 .. I am working in webdriverio/webdriverio#12400 to improve this weirdness a bit

Reply:
Hmm, I completely need to overwrite the caps, will see how I can do that with multiremote then

Reply:
throwing an error for multiremote, there's no easy way to fix this